static/app/views/releases/thresholdsList/thresholdGroupRows.tsx

```diff
@@ -91,15 +91,53 @@ export function ThresholdGroupRows({
       },
     ];
     if (isInternal) {
-      list.push({
-        value: CRASH_FREE_SESSION_RATE_STR,
-        textValue: 'Crash Free Sessions',
-        label: 'Crash Free Sessions',
-      });
+      list.push(
+        {
+          value: CRASH_FREE_SESSION_RATE_STR,
+          textValue: 'Crash Free Sessions',
+          label: 'Crash Free Sessions',
+        },
+        {
+          value: NEW_ISSUE_COUNT_STR,
+          textValue: 'New Issue Count',
+          label: 'New Issue Count',
+        }
+      );
     }
     return list;
   }, [organization]);
 
+  const windowOptions = thresholdType => {
+    let options = [
+      {
+        value: 'hours',
+        textValue: 'hours',
+        label: 'hrs',
+      },
+      {
+        value: 'days',
+        textValue: 'days',
+        label: 'days',
+      },
+    ];
+    if (thresholdType !== CRASH_FREE_SESSION_RATE_STR) {
```

Comment:
is it not possible to specify minute resolution when querying for crash free sessions? are we query snuba for this?

Reply:
Yea unfortunately it wouldn't make sense for this application

If we drop the interval resoultion too much, then we'd be getting back far too many results and the UI just wouldn't work as expected

Reply:
So decision here was that we'd just default to 1hr resolution minimum.
for now it's fine and we can re-address this at a later point after we receive feedback